doc/env_vars.md

```diff
@@ -87,6 +87,7 @@ You may have noted that some options are not available:
 | `LRSQL_AUTHORITY_URL` | `authorityUrl` | The URL that is set as the `authority-url` value when constructing an authority from a template. | `http://example.org` |
 | `LRSQL_OIDC_AUTHORITY_TEMPLATE` | `oidcAuthorityTemplate` | Like `LRSQL_AUTHORITY_TEMPLATE`, but only used when forming an authority from an OIDC access token. | <details>`config/oidc_authority.json.template`<summary>(Filepath)</summary></details> |
 | `LRSQL_OIDC_SCOPE_PREFIX` | `oidcScopePrefix` | An optional prefix prepended to OIDC scope. For example, setting this to `lrs:` would change the expected `all` scope to `lrs:all` | `""` |
+| `LRSQL_OIDC_CLIENT_ID` | `oidcClientId` | An optional OIDC client ID for the SQL LRS Admin SPA. If provided, along with the `LRSQL_OIDC_ISSUER` and `LRSQL_OIDC_AUDIENCE` webserver variables, will enable OIDC access to the Admin UI. | Not Set |
```

Comment:
Two things:
- There's no documentation for `:oidc-client-template` yet.
- Why did you put `:oidc-client-id` under `lrs` and not `webserver`? Especially since it directly relates to the admin UI, which is generally controlled by webserver variables, and both the other "OIDC Enabler Vars" (`:oidc-issuer` and `:oidc-audience`) are webserver vars as well.

Reply:
> There's no documentation for :oidc-client-template yet.

Reply:
On the location of `:oidc-client-id`, I based it on the presence of other domain-level admin concerns there like the authority templates + admin defaults. That said I don't really care where it goes and am happy to change it

Reply:
Likewise I think it should go in the webserver section for similar reasons as above.

Reply:
sure, whatever

Reply:
https://github.com/yetanalytics/lrsql/pull/210/commits/202b4a079b75359b18eff37d3e527c09a4c80acc